tests/Nox.Types.Tests/Types/MacAddress/MacAddressTests.cs

```diff
@@ -3,9 +3,117 @@ namespace Nox.Types.Tests.Types;
 
 public class MacAddressTests
```

Comment:
According to the latest info, test should be written using FluentAssertions. Can you please add the required changes here? 

https://regusit.atlassian.net/wiki/spaces/NOX/pages/3153264693/Test+Projects

You can find sample implementation here: https://github.com/NoxOrg/Nox.Generator/blob/main/tests/Nox.Types.Tests/Types/Volume/VolumeTests.cs